src/bcccp/carpark/paystation/PaystationController.java

```diff
@@ -47,9 +47,9 @@ public void ticketPaid() {
 
   @Override
   public void ticketTaken() {
```

Comment:
yeah i'm not actually responsible for paystation controller, that is chris' class.